resources/views/scenario/view.blade.php

```diff
@@ -0,0 +1,40 @@
+{{-- Add the layout from the user login setup here. --}}
+{{--@extends('layouts.app')--}}
+
+{{--@section('content')--}}
+    {{-- container to contain the scenario --}}
+    <div class="container">
+        {{-- card for scenario --}}
+        <div class="card">
+            <div class="card-header">
+                <h2 class="card-title">Casus: {{$scenario->story->title}}</h2>
+            </div>
+            <div class="card-body">
+                <h2 class="card-title"> {{$scenario->id}}</h2>
```

Comment:
Is het belangrijk om het scenario id te laten zien op de pagina?

Reply:
r. 13 verwijdert uit code.